bacon.xcodeproj/project.pbxproj

```diff
@@ -425,7 +428,7 @@
 			inputFileListPaths = (
 			);
 			inputPaths = (
-				"${SRCROOT}/Pods/Target Support Files/Pods-bacon/Pods-bacon-frameworks.sh",
+				"${PODS_ROOT}/Target Support Files/Pods-bacon/Pods-bacon-frameworks.sh",
```

Comment:
I'm not sure if this should be committed o.o I didn't commit this change even though the same change got highlighted o.o @GilgameshTC any idea?